crypto/x509v3/v3_ncons.c

```diff
@@ -9,6 +9,7 @@
 
 #include <stdio.h>
 #include "internal/cryptlib.h"
+#include "e_os.h"
```

Comment:
As for separate header. It's fine line. In sense that we don't really want too many of them. At least we don't want as many as standard ones, do we? And considering this example it might be tricky. Note that it started with strncasecmp, but ended with suggestion to create internal/string.h. See the contradiction? strncasecmp is defined strings.h, not string.h... I would like to suggest to proceed with this request without creating new files, collect our thoughts and do something about it in separate request. [To tell a little secret I was going to discuss this already at first meeting, but agenda was jammed.]